src/item.cpp

```diff
@@ -3542,6 +3565,60 @@ bool item::is_artifact() const
     return type->artifact.get() != nullptr;
 }
 
+bool item::can_contain( const item &it ) const
+{
+    // @todo Volume check
+    return can_contain( *it.type );
+}
+
+bool item::can_contain( const itype &tp ) const
+{
+    if( type->container == nullptr ) {
+        // @todo: Tools etc.
+        return false;
+    }
+
+    if( tp.phase == LIQUID && !type->container->watertight ) {
+        return false;
+    }
+
+    // @todo Acid in waterskins
+    return true;
+}
+
+bool item::spill_contents( Character &c )
+{
+    if( c.is_npc() ) {
+        spill_contents( c.pos() );
```

Comment:
The return value of that function call is not used, is that intended? Should this return right here and skip the remaining parts of the function?